source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -126,6 +128,22 @@
 
 USE_TZ = True
 
+LANGUAGES = [
+    ('en', _('English')),
+    ('fr', _('French')),
+]
+
+# Do not import the real utils here! We don't want an infinite loop
+gettext = lambda x: x
```

Comment:
It's just a question : What the purpose of this line ?

Reply:
It should not be there. In fact, to avoid the circular importation, we need to use ugettext_lazy here.